diff --git a/Tester/main.c b/Tester/main.c
index 3519687..fb08b61 100755
--- a/Tester/main.c
+++ b/Tester/main.c
@@ -430,7 +430,7 @@ int main(int argc, char **argv)
         GB_set_rgb_encode_callback(&gb, rgb_encode);
         GB_set_log_callback(&gb, log_callback);
         GB_set_async_input_callback(&gb, async_input_callback);
-        GB_set_color_correction_mode(&gb, GB_COLOR_CORRECTION_EMULATE_HARDWARE);
+        GB_set_color_correction_mode(&gb, GB_COLOR_CORRECTION_DISABLED);
         GB_set_rtc_mode(&gb, GB_RTC_MODE_ACCURATE);
         GB_set_emulate_joypad_bouncing(&gb, false); // Adds too much noise
         
